### DISCUSSION TITLE
关于屏幕取词操作的建议

### DISCUSSION THREAD

**Discussion by mustang88** [xiaoyifang/goldendict-ng#1610] [Ideas]
目前NG版本开启屏幕取词后，只要是复制操作就会引发取词。哪怕是修改了快捷键也与此无关。
建议：
在屏幕取词的功能处，设置取词方式的选择项，可以是目前的复制操作，也可以是其他由用户自定义的功能键组合才引发，这样就不至于正常处理文字时因复制与粘贴操作而引起取词操作。

**Comment by xiaoyifang** [comment]
![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/956c682d-1781-4077-9cd2-ef17917f9bb3)


把这个关掉，用默认的Ctrl+C+C取词，应该就可以达到你的目的了。
